pixeltable/exprs/expr.py

```diff
@@ -1,21 +1,25 @@
 from __future__ import annotations
+
 import abc
+import hashlib
+import importlib
+import inspect
+import json
 import sys
 import typing
+from itertools import islice
 from typing import Union, Optional, List, Callable, Any, Dict, Tuple, Set, Generator, Type
-import json
 from uuid import UUID
-import hashlib
-import importlib
 
 import sqlalchemy as sql
 
-from .globals import ComparisonOperator, LogicalOperator, LiteralPythonTypes, ArithmeticOperator
-from .data_row import DataRow
 import pixeltable
-import pixeltable.exceptions as excs
 import pixeltable.catalog as catalog
+import pixeltable.exceptions as excs
 import pixeltable.type_system as ts
+from pixeltable.func import make_function
+from .data_row import DataRow
```

Comment:
I'm using "optimize imports" in PyCharm to put them in a canonical order. It's pretty standard and (I think) it's also the ordering that linters expect: PEP8 ordering for import categories, alphabetical within each category. It will also remove unused imports.